src/selfplay/game.cc

```diff
@@ -69,7 +69,15 @@ void SelfPlayGame::Play(int white_threads, int black_threads) {
     // Add best move to the tree.
     Move move = search_->GetBestMove().first;
     tree_[0]->MakeMove(move);
-    if (tree_[0] != tree_[1]) tree_[1]->MakeMove(move);
+    if (!reuse_tree_) {
```

Comment:
Actually, I think it makes sense to make it per player options, match "with tree reuse  vs no tree reuse" would totally make sense.

Also, I'd call TrimTreeAtHead() before the search, and only for the player which is about to start thinking.

Reply:
I have no clue how to make a per-player option :)

Reply:
It's already in `options_[idx].uci_options.Get<bool>("Reuse the node statistics between moves.")`, which is per player, but the question is how to share that string constant "Reuse the node statistics between moves." nicely between tournament.cc and game.cc.

The cleanest way would be to have static function `SelfPlayGame::PopulateOptions`, and call it from `SelfPlayTournament::PopulateOptions`, and define that constant in game.cc rather than tournament.cc. But feel free to solve it differently if you wish.

Reply:
I think I've worked it out, maybe...